extension/app/ts/components/pages/InterceptorAccess.tsx

```diff
@@ -116,17 +116,24 @@ interface InterceptorAccessRequest {
 	socket: WebsiteSocket
 }
 
+const DISABLED_DELAY_MS = 3000
```

Comment:
I started to write a comment about how we should do this, but I'm happy to see you already thought of it!